styleframe/style_frame.py

```diff
@@ -374,6 +387,8 @@ def to_excel(self, excel_writer: Union[str, pd.ExcelWriter, pathlib.Path] = 'out
                       calling ``StyleFrame.to_excel`` by directly modifying ``StyleFrame.A_FACTOR`` and ``StyleFrame.P_FACTOR``
 
         :type best_fit: None or str or list or tuple or set
+        :param index: Write row names.
+        :type index: bool
```

Comment:
I suggest to merge the param name and type

```suggestion
        :param bool index: Write row names.
```